docs/cli/releases.mdx

```diff
@@ -120,22 +120,6 @@ sentry-cli releases set-commits "$VERSION" --auto --ignore-missing
 
 When you are working with JavaScript and other platforms, you can upload release artifacts to Sentry which are then considered during processing. The most common release artifact are [source maps](/platforms/javascript/sourcemaps/) for which `sentry-cli` has specific support.
```

Comment:
This snippet should also be changed to reflect the deprecation, or perhaps, we should remove it since I don't think it makes a ton of sense.

```suggestion
```